operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -191,62 +213,81 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
             throw new Error("There has been an error retrieving the list of included resources of the $everything operation.", e);
         }
         
+        Map<String, ParametersMap> supportedSearchParameters = ParametersUtil.getTenantSPs(FHIRRequestContext.get().getTenantId());
         List<String> resourceTypesOverride = getOverridenIncludedResourceTypes(parameters, defaultResourceTypes);
         List<String> resourceTypes = resourceTypesOverride.isEmpty() ? defaultResourceTypes : resourceTypesOverride;
-
-        int totalResourceCount = 0;
+ 
+        boolean retrieveAdditionalResources = extraResources != null && !extraResources.isEmpty();
         for (String compartmentType : resourceTypes) {
```

Comment:
i know its not from you, but `compartmentType` still trips me up here.  maybe change to `compartmentMemberType` for consistency with the other part I had you change?

Reply:
Done